chipy/videos/in-project-virtualenvs.json

```diff
@@ -15,10 +15,9 @@
   "summary": "",
   "tags": [],
   "thumbnail_url": "http://i1.ytimg.com/vi/kXoSASlq7zU/hqdefault.jpg",
-  "title": " In-project virtualenvs ",
+  "title": "In-project virtualenvs ",
```

Comment:
Looks like trailing whitespace were missed.
Is it possible to strip these too?

Reply:
Thanks, I will look into it.